app/views/idv/doc_auth/ssn.html.erb

```diff
@@ -1,5 +1 @@
-<% if flow_session['pii_from_doc']['ssn'] %>
-  <%= render 'idv/doc_auth/ssn_update', flow_session: flow_session %>
-<% else %>
-  <%= render 'idv/doc_auth/ssn_init', flow_session: flow_session, should_render_id_success_alert: true %>
-<% end %>
+<%= render 'idv/shared/ssn', flow_session: flow_session, id_success_alert_enabled: !applicant_ssn, updating_ssn: applicant_ssn  %>
```

Comment:
would applicant_ssn ever be the empty string? if so should we so something like: 
```suggestion
<%= render 'idv/shared/ssn', flow_session: flow_session, id_success_alert_enabled: applicant_ssn.blank?, updating_ssn: applicant_ssn  %>
```

Reply:
Make sense, added in da36d11064149805774a30684df69fbc3e1a06aa